### PR TITLE
Build documentation improvement

### PR DESCRIPTION
**Pull request by orionlee**
1. Include all dependencies in a `docs/requirements.txt`, and update the instructions.
2. Make windows build behave the same as Linux/Mac.

### CODE REVIEW

File: docs/source/about/documentation.rst [hunk 1]
```diff
@@ -20,18 +20,14 @@ Building documentation
...
-- sphinx
-- sphinx-automodapi
-- sphinxcontrib-rawfiles
-- nbsphinx
-- ghp-import
-- graphviz
-- `numpydoc <https://github.com/numpy/numpydoc>`_
-
-These packages can be installed using `conda` or `pip`.
+Building the *lightkurve* documentation requires a `sphinx` and few extra packages. To install them::
```
Comment (barentsen) [inline docs/source/about/documentation.rst:23]: Fixing a typo:
```suggestion
Building the *lightkurve* documentation requires `sphinx` and a few extra packages. To install them::
```

**Review by barentsen — APPROVED** [review]
Thank you @orionlee. This is a great improvement, once again! :+1:

I hope you're signed up for [Hacktoberfest](https://hacktoberfest.digitalocean.com/) =)

**Comment by orionlee** [comment]
I forgot about one thing: in my local doc build, I cannot get the class inheritance diagrams rendered. I don't know if it's a Windows-specific issue, or there are some other packages needed that are not listed.

Examples on how the doc was shown on my local build:

image

**Comment by orionlee** [comment]
@barentsen is `Lightkurve` associated with Hacktoberfest in anyway?

**Comment by barentsen** [comment]
@orionlee You sign up as an individual to get a t-shirt.  I believe it doesn't matter which repository your make PR's in, but we've definitely received t-shirts for making Lightkurve PRs in previous years!

**Pull request merged**